### DISCUSSION TITLE
Will you guys ever accept BTC for donations?

### DISCUSSION THREAD

**Discussion by TheGuySwann** [transmission/transmission#3999] [General]
**Short version:** 
• I would've donated a few dollars worth on numerous occasions if the option was available because its what I use every day and is easiest & most accessible for me.

**Long version:** 
• I have everything setup through BTC and Lightning payments in my browser and its shockingly easy for me to donate to projects, throw a few bucks to people for assistance with some software and/or help in finding sources for a story, etc, and I do this literally every day. 

That said, I've been notified to donate to Transmission a number of times over the years, and every time I click on it, look for BTC, and then don't donate because the only option is Paypal (which is horrible) or giving my credit card info, which seems antithetical to community principles and safety of users (especially depending on their jurisdiction).

Accepting BTC on chain and via lightning is really easy and would solve these issues and there are numerous services (like OpenNode or Strike) that make this incredibly easy, or you can host your own BTCpayserver (simple way is super fast setup & hosting with a service like Voltage) and be your own processor.

Anyway, I've just been donating to a lot of projects lately when i easily click and invoice and keep going. Its fast and i don't have to go looking for my wallet or connect to some company that's recording my transaction and sending all my private information to 10 new companies. 

Would be nice to throw some sats at Transmission every now and then as well.

**Comment by Sjors** [comment]
Ditto. I deleted my PayPal a long time ago, and in general I prefer not to disclose enormous amounts of person information when making a donation - which is what happens when you pay by creditcard.

There's an even easier (non-custodial) way now: install the Phoenix app, click Receive, click "Show reusable QR", copy the string that starts with `lno1` and post that on the donation page. That should Just Work(tm).

(even better privacy if you put it in the application, so you don't even get the IP address of a potential donor).
